config/mocha.browser.opts

```diff
@@ -0,0 +1,2 @@
+--timeout 20000
```

Comment:
should we add the google license as in the file you deleted?

Reply:
I think only js/ts files require a license header? Js configuration files under the 'config' directory do have a license header, while all the others don't. But I'm not sure. Your call.

P.S. Mocha version needs an upgrade to support comments in mocha config file, if we decide to add a header.